examples/seismic/tti/tti_example.py

```diff
@@ -21,7 +21,7 @@ def tti_setup(shape=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=250.0,
     return AnisotropicWaveSolver(model, geometry, space_order=space_order, **kwargs)
 
 
-def run(shape=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=250.0,
+def run(shape=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=500.0,
```

Comment:
I don't like this bump to 500. I don't see why it's necessary. I'm used to the turnaround time of `python tti_example.py` which I find not too short not too long, doubling it would make it slow for no evident reason

Reply:
`tn` was set again to be 250.0. It only is changed in the specific test as you suggested.